sensors/sensors.go

```diff
@@ -46,37 +38,30 @@ type TimedSensor interface {
 func NewLidar(
 	ctx context.Context,
 	deps resource.Dependencies,
-	sensors []string,
+	cameraName string,
 	logger golog.Logger,
 ) (Lidar, error) {
 	_, span := trace.StartSpan(ctx, "viamcartographer::sensors::NewLidar")
 	defer span.End()
-
-	// An empty `sensors: []` array is allowed in offline mode.
-	if len(sensors) == 0 {
-		logger.Debug("no sensor provided in 'sensors' config parameter")
-		return Lidar{}, nil
-	}
-	// If there is a sensor provided in the 'sensors' array, we enforce that only one
-	// sensor has to be provided.
-	if len(sensors) != 1 {
-		return Lidar{}, errors.Errorf("configuring lidar camera error: "+
-			"'sensors' must contain only one lidar camera, but is 'sensors: [%v]'",
-			strings.Join(sensors, ", "))
-	}
-
-	name, err := getName(sensors, lidarIndex)
+	newLidar, err := camera.FromDependencies(deps, cameraName)
 	if err != nil {
-		return Lidar{}, err
+		return Lidar{}, errors.Wrapf(err, "error getting lidar camera %v for slam service", cameraName)
 	}
 
-	newLidar, err := camera.FromDependencies(deps, name)
-	if err != nil {
-		return Lidar{}, errors.Wrapf(err, "error getting lidar camera %v for slam service", name)
-	}
+	// To be implemented once replay camera supports Properties
```

Comment:
Is there a ticket for this? If, so please add the ticket URL to the top of this comment. If not, please create a ticket (in the IMU project epic) and put the URL to it at the top of this comment.

Reply:
Added this to the ticket for movement sensor replay